### ISSUE TITLE
Cannot find module "localforage" (Ionic 3.8.0)

### ISSUE THREAD

**Issue by sanderaarts** [ionic-team/ionic-storage#128]
Since updating to Ionic 3.8.0 I'm getting a `Cannot find module "localforage"` error. The module is available (`node_modules/@ionic/storage/node_modules/localforage`) but is not found. If I manually install it (`node_modules/localforage`) it is found, but then I get the same for `localforage-cordovasqlitedriver`.

> ionic (Ionic CLI) : 3.16.0
Ionic Framework : ionic-angular 3.8.0
Node : v8.9.0
npm  : 5.5.1 
OS : OS X Yosemite

**Comment by ionicpremium** [comment]
Same here :(

**Comment by borodiliz** [comment]
Same problem here after upgrading `npm` from `4.6.1` to `5.5.1`
Downgrading `npm` to 4.6.1 solves the problem :confused: :
```
npm install -g npm@4.6.1
rm -rf node_modules
npm install
```

**Comment by fishme** [comment]
@borodiliz you are my hero

**Comment by edmundo096** [comment]
Landed here after Googling the same issue a bit (just referenced the first similar results above).

I found out that **I forgot to remove** `package-lock.json` just after updating ionic from `3.7.0` to `3.9.0` and removing and re-installing `node_modules`.
This is mentioned on the [CHANGELOG.md](https://github.com/ionic-team/ionic/blob/master/CHANGELOG.md#390-2017-11-08) file as well, I just missed that step.

Updating `npm` from `5.5.1` that I had to `5.6.0` didn't worked, but was because my error described above.
